extensions/wikia/AdEngine/js/tracking/adInfoListener.js

```diff
@@ -91,9 +91,9 @@ define('ext.wikia.adEngine.tracking.adInfoListener',  [
 		}
 
 		tracker.track(
-			vastInfo.pos,
-			vastInfo,
-			vastInfo,
+			vastInfo.customParams.pos,
+			vastInfo.customParams,
```

Comment:
Does line item id / creative id / media type / etc (all info about played ad video) still working after this change?

Reply:
I believe that this argument (second) should stay `vastInfo`. Please check some test page with targeted preroll:
project43.wikia.com/wiki/SyntheticTests/Premium/FeaturedVideo

Reply:
Yep, everything is fine.

Reply:
Oh, got it. Sorry!